src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -277,9 +277,29 @@ private static SystemEnforcementMode GetAppLockerPolicy(string path, SafeHandle
                     }
                     finally
                     {
-                        if (IO.File.Exists(testPathScript)) { IO.File.Delete(testPathScript); }
+                        if (IO.File.Exists(testPathScript))
+                        {
+                            try
+                            {
+                                IO.File.Delete(testPathScript);
+                            }
+                            catch
```

Comment:
Maybe instead of catching everything, catch only IOException?
Same for below.

Reply:
And I'd move this dup code in helper method `TryDeleteFile()`.

Reply:
Will update the code based on suggestions